build/charts/yorkie-cluster/templates/istio/ratelimit/config.yaml

```diff
@@ -0,0 +1,15 @@
+{{ if .Values.ratelimit.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ratelimit-config
+  namespace: {{ .Values.yorkie.namespace }}
```

Comment:
How about moving ratelimit resources to `istio` namespace? Since ratelimit is applied to ingressgateway in `istio` namespace, it will be better to place them in `istio` namespace.